How to Find and Use Technical Resources if You’re New to Tech:proposal.md

```diff
@@ -0,0 +1,60 @@
+# Outline - "How to Find Coding Resources if You're New to Tech"
+
+### Step One
+
+**Title:**
+
+How to Find Coding Resources if You're New to Tech
+
+**Draft:**
+Why not dip your toes into the world of coding if you have some extra time on your hands? Whether you're bored during summer or want to learn new skills, learning a bit of coding is a great idea. Knowing how to code is invaluable knowledge, not matter what level of mastery you're at. And as a bonus, it gives you the ability to build and create awesome projects with more flair and creativity. 
+  
+**What is coding?**
+
+This might seem like a basic question, but if you're brand new to coding, its good to go over the basics! In sum, coding is using programming languages to tell a computer how to behave. As a coder, you're using the computer's language to make it do something. 
+
+This could be as simple as making your font bigger and bolder, or it could be more complicated, like coding a video game. 
+
+**What are some coding languages?**
+
+You've probably already heard of some languages that coders and progammers use in their work: HTML, CSS, Javascript, Python, etc. 
+
+When I was first introduced to coding, I learned HTML and CSS. HTML is sort of like the bare bones of the internet; it displays documents in web browsers. CSS is a stylesheet that affects how HTML is displayed, such as fonts, colors, formats. Essentially, HTML dictates what the content is, and CSS dictates how that content looks.  
+
+These weren't too difficult to learn, and if you're not sure where to start, you can start here  to get a better understanding of coding in general. (Sidenote: HTMl is actually a markup language because it doesn't have any programming logic like other languages.)
+
+But if you really want to become more advanced with coding, languages like Javascript and Python could be more up your alley. Javascript makes web designs more interactive, which engages users more than just HTML and CSS. Python is often used for creating web applications, and it is versatile for both small and big projects. 
+
+Depending on what you want to learn, you might better enjoy learning some languages over others. Take a couple minutes to research and gain a better understanding of what skills you hope to build before diving straight into learning. 
+
+**Taking coding lessons with freeCodeCamp**
+ 
+ One great place for you to start as a newbie to coding is [freeCodeCamp](https://www.freecodecamp.org/). This nonprofit organization helps users code through hands-on projects and structured, easy-to-manage lessons. 
+
+In addition to their learning platform, they also have Youtube videos and can connect you to a community of others who are just as excited about coding as you are. 
+
+If you're feeling particularly ambitious, they offer certification programs too in topics like responsive web design and Javascript algorithims and data structures. 
+  
+**Putting your new coding skills to work**
```

Comment:
I also think another idea could be to rank online resources (based on which ones offer more personalized learning for example). Maybe when interviewing programmers, you could ask which resources they found the most helpful. I think it would give more context into why an online resource was mentioned in the blog.